tests/integration/src/test/java/org/apache/pulsar/tests/integration/PulsarStandAloneContainer.java

```diff
@@ -0,0 +1,121 @@
+package org.apache.pulsar.tests.integration;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.command.LogContainerCmd;
+import com.github.dockerjava.api.model.Frame;
+import com.github.dockerjava.core.command.LogContainerResultCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+public class PulsarStandAloneContainer extends GenericContainer<PulsarStandAloneContainer> {
```

Comment:
We shouldn't use standalone in integration tests as it doesn't represent the software that runs on production clusters. Except of course to test that standalone boots and runs, which we should be testing.

Reply:
that's what's it meant for. We will providing a variety of cluster object templates. To check different conditions.